code/modules/mob/living/simple_animal/bot/honkbot.dm

```diff
@@ -196,7 +196,9 @@
 	if(spam_flag == 0)
 		if(ishuman(C))
 			C.stuttering = 20
-			C.adjustEarDamage(0, 5) //far less damage than the H.O.N.K.
+			var/obj/item/organ/ears/ears = C.getorganslot(ORGAN_SLOT_EARS)
+			if (ears && !HAS_TRAIT_FROM(C, TRAIT_DEAF, CLOTHING_TRAIT))
```

Comment:
In general rather than checking if we are deaf from clothing, we should check the total ear protection of that mob. Clothing should provide ear protection without making the mob entirely deaf and any clothing that will make the mob deaf should provide maximum hearing protection anyway